### ISSUE TITLE
Build is broken

### ISSUE THREAD

**Issue by taybin** [node-influx/node-influx#352]
```
src/pool.ts(321,39): error TS2345: Argument of type '<T>(arg: T) => any' is not assignable to parameter of type '() => void'.
```

Additionally, there are a ton of lint errors when running `npm run test:lint`.  Are those known?

Running nodejs 11.12.0.

**Comment by taybin** [comment]
https://github.com/node-influx/node-influx/pull/353

**Comment by bencevans** [comment]
Thanks for your report. Build issues are now fixed and into `master` from #353.
